test/javascripts/acceptance/chat-browse-test.js

```diff
@@ -191,13 +202,21 @@ acceptance("Discourse Chat - chat browsing no channels", function (needs) {
     assert.ok(exists(".start-creating-dm-btn"));
 
     await click(".start-creating-dm-btn");
-    assert.ok(exists(".dm-creation-row"));
-    let users = selectKit(".dm-user-chooser");
-    await click(".dm-user-chooser");
-    await users.expand();
-    await fillIn(".dm-user-chooser input.filter-input", "hawk");
-    await users.selectRowByValue("hawk");
-    await click("button.create-dm");
-    assert.equal(currentURL(), "/chat/channel/75/@hawk");
+
+    assert.equal(
+      currentURL(),
+      `/chat/channel/draft/${encodeURIComponent(
+        I18n.t("chat.direct_message_creator.title")
```

Comment:
Yes mostly this is a kind of edge case to avoid a dedicated route. I'm mixed on this. The big pros is that it avoids a lot of code, but it can feel slightly hacky I agree.

Reply:
I tend to keep it that way for now, but we can think on how to make this better with mark.